src/ray/gcs/gcs_client/service_based_gcs_client.cc

```diff
@@ -50,7 +50,12 @@ Status ServiceBasedGcsClient::Connect(boost::asio::io_service &io_service) {
                                               *client_call_manager_, get_server_address));
 
   job_accessor_.reset(new ServiceBasedJobInfoAccessor(this));
-  actor_accessor_.reset(new ServiceBasedActorInfoAccessor(this));
+  if (getenv("RAY_GCS_SERVICE_ACTOR_MANAGEMENT_ENABLED") != nullptr) {
+    RAY_CHECK(getenv("RAY_GCS_SERVICE_ENABLED") != nullptr);
```

Comment:
seems a bit strange to have two environment variables here?

Reply:
Yes, the `RAY_GCS_SERVICE_ACTOR_MANAGEMENT_ENABLED` must along with `RAY_GCS_SERVICE_ENABLED`, but there will be only one environment 
 of `RAY_GCS_SERVICE_ENABLED` in the future.